packages/webapp/src/containers/saga.js

```diff
@@ -619,6 +620,17 @@ export function* fetchAllSaga() {
 export function* clearOldFarmStateSaga() {
   yield put(resetTasks());
 
+  yield put(
+    api.util.invalidateTags([
+      'Animals',
+      'AnimalBatches',
+      'AnimalGroups',
+      'CustomAnimalBreeds',
+      'CustomAnimalTypes',
+      'DefaultAnimalTypes', // needs to be cleared for KPI count
+    ]),
+  );
```

Comment:
Since almost all of the data in the app is dependent on the farm, I wonder if we should consider calling `api.util.resetApiState()` instead?

Reply:
Well it would result in re-fetching the enum keys and the default breeds unnecessarily. But the advantage is, it would be a set it and forget it type situation 🤔

Reply:
That's a good point!